_data/Biospheere/Biospheere.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Niklas Marion
+institution: Saarlouiser Gymnasium am Stadtgarten
+profile_pic: niklas.jpg
+quote: Code is like humor. When you have to explain it, it’s bad.
```

Comment:
Change non-ascii character ’ to single quote '.

Reply:
Removed the non-ascii character. Is it okay like this?